src/main/java/com/aws/greengrass/secretmanager/SecretManager.java

```diff
@@ -132,11 +153,21 @@ public void syncFromCloud(List<SecretConfiguration> configuredSecrets) throws Se
                 } catch (IOException | SdkClientException e) {
                     AWSSecretResponse secretFromDao = secretDao.get(secretArn, label);
                     if (secretFromDao != null) {
-                        downloadedSecrets.add(secretFromDao);
-                        logger.atDebug().kv("secret", secretArn).kv("label", label)
-                                .log("Secret configuration is not changed. Loaded from local store");
                         logger.atWarn().kv("secret", secretArn).kv("label", label)
-                                .log(String.format("Could not sync secret %s, label %s", secretArn, label));
+                                .log("Could not sync secret from cloud, but we have a local version which may work");
+                        // We couldn't sync it from the cloud, try loading it from local copy
+                        try {
+                            // Ensure that we're able to decrypt it. If we are unable to decrypt it
+                            // then we have no more fallbacks and the customer needs to fix the issue.
+                            decrypt(secretFromDao);
```

Comment:
Lets modify the log to tell the customer how to remediate. Customer can retry the fetch by re deploying secret component if connectivity is back.

Reply:
What wording would you suggest?

Reply:
I just want to inform them that they can attempt refetch on re deployment. May be "Could not download secret from cloud, you could attempt a re fetch by re deploying secret manager"